package.json

```diff
@@ -69,7 +69,7 @@
     "remixicon": "^4",
     "sharp": "^0",
     "viem": "^2",
-    "wagmi": "^2",
+    "wagmi": "2.5.7",
```

Comment:
Oh another one, can you add back the `^` there? so Wagmi can properly update in the range of the current major version (2) when we run `bun update`